arm-servicebus/2015-08-01/swagger/eventhub.json

```diff
@@ -0,0 +1,1862 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "EventHubManagementClient",
+    "description": "Azure EventHub client",
+    "version": "2014-09-01"
```

Comment:
Sorry, the changes were not made to the EH swagger spec. I am removing it from the PR until all the right changes are made. You can review the changes to SB swagger spec.